frontend/src/screens/ProductScreen.tsx

```diff
@@ -63,18 +52,20 @@ export const ProductScreen = ({ history, match }: ProductScreenProps) => {
           <Col md={6}>
             <Image
               className="rounded"
-              src={product.image}
-              alt={product.name}
+              src={product.getProduct.image}
+              alt={product.getProduct.name}
```

Comment:
product.getProduct looks weird…

Instead of doing `data: product`, do:


```ts
const product = data.getProduct
```

Reply:
done